pkgs/by-name/an/anytype/package.nix

```diff
@@ -17,7 +17,8 @@ in appimageTools.wrapType2 {
   extraInstallCommands = ''
     source "${makeWrapper}/nix-support/setup-hook"
     wrapProgram $out/bin/${pname} \
-      --add-flags "\''${NIXOS_OZONE_WL:+\''${WAYLAND_DISPLAY:+--ozone-platform-hint=auto --enable-features=WaylandWindowDecorations}}"
+      --add-flags "\''${NIXOS_OZONE_WL:+\''${WAYLAND_DISPLAY:+--ozone-platform-hint=auto --enable-features=WaylandWindowDecorations}}" \
+      --add-flags ${lib.escapeShellArg commandLineArgs}
```

Comment:
Have you tried with empty `commandLineArgs`?

Reply:
@Aleksanaa 

I've tried it, it looks like this below:

```bash
#! /nix/store/izpf49b74i15pcr9708s3xdwyqs4jxwl-bash-5.2p32/bin/bash -e
exec -a "$0" "/nix/store/8b47mww94ggxy5nh4zznzw106bs90s6h-anytype-0.42.8/bin/.anytype-wrapped"  ${NIXOS_OZONE_WL:+${WAYLAND_DISPLAY:+--ozone-platform-hint=auto --enable-features=WaylandWindowDecorations}}  "$@" 
```

if I use ` commandLineArgs = "--enable-wayland-ime"; `, it looks like:
```bash
#! /nix/store/izpf49b74i15pcr9708s3xdwyqs4jxwl-bash-5.2p32/bin/bash -e
exec -a "$0" "/nix/store/3n3vj94sj43h6ykg7wrmawy9cz0bmsp9-anytype-0.42.8/bin/.anytype-wrapped"  ${NIXOS_OZONE_WL:+${WAYLAND_DISPLAY:+--ozone-platform-hint=auto --enable-features=WaylandWindowDecorations}} --enable-wayland-ime "$@" 
```